doc/examples/animation_widget/lib/main.dart

```diff
@@ -9,6 +9,7 @@ import 'package:flutter/material.dart';
 Sprite _sprite;
 
 void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
```

Comment:
do we still need this? I thought this was not necessary anymore, does this example give out an error w/o?

Reply:
(if this is unrelated to your change it's fine)

Reply:
I get the following error if run without `ensureInitialized()`:
```
[VERBOSE-2:ui_dart_state.cc(157)] Unhandled Exception: ServicesBinding.defaultBinaryMessenger was accessed before the binding was initialized.
If you're running an application and need to access the binary messenger before `runApp()` has been called (for example, during plugin initialization), then you need to explicitly call the `WidgetsFlutterBinding.ensureInitialized()` first.
If you're running a test, you can call the `TestWidgetsFlutterBinding.ensureInitialized()` as the first line in your test's `main()` method to initialize the binding.
#0      defaultBinaryMessenger.<anonymous closure> (package:flutter/src/services/binary_messenger.dart:76:7)
#1      defaultBinaryMessenger (package:flutter/src/services/binary_messenger.dart:89:4)
#2      PlatformAssetBundle.load (package:flutter/src/services/asset_bundle.dart:219:15)
#3      Images._fetchToMemory (package:flame/images.dart:30:46)
#4      Images.load (package:flame/images.dart:24:37)
#5      Sprite.loadSprite (package:flame/sprite.dart:47:44)
<…>
```
I'm on Flutter dev channel.